seonshine_frontend/src/pages/userManagement/components/UserManagementTab/index.tsx

```diff
@@ -0,0 +1,125 @@
+import { useState } from 'react';
+import { toast } from 'react-toastify';
+
+import { Stack, Typography } from '@mui/material';
+import { useQueryClient } from '@tanstack/react-query';
+
+import SearchBar from '@/components/molecules/searchBar';
+import ConfirmModal from '@/components/organims/confirmModal';
+import { userType } from '@/components/organims/sideBar';
+import Table from '@/components/organims/table';
+import UserProfileModal from '@/components/organims/userProfileModal';
+
+import useTable from '@/hooks/useTable';
+import { ChangeStatusPayloadType, UserStatusEnum, UserType } from '@/types/user';
+
+import { useChangeStatusApi, useGetUserListApi } from '@/apis/hooks/userApi.hook';
+
+import { activeUserDescription, activeUserTitle, deactiveUserDescription, deactiveUserTitle } from './constants';
+import { UserTableHeader } from './UserTableHeader';
+
+const ITEMS_PER_PAGE = 5;
+
+const UserManagementTab = () => {
+  const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
+  const [isConfirmModalOpen, setIsConfirmModalOpen] = useState<boolean>(false);
+  const [selectedUser, setSelectedUser] = useState<ChangeStatusPayloadType>();
+
+  const queryClient = useQueryClient();
+
+  const {
+    currentPage,
+    sortKey,
+    sortType,
+    pageSize,
+    handlePageChange,
+    handleSortingChange,
+    searchField,
+    searchQuery,
+    handleSearchChange,
+  } = useTable(ITEMS_PER_PAGE);
+
+  const { data, isFetching } = useGetUserListApi({
+    page_size: pageSize,
+    page_number: currentPage,
+    sort_key: sortKey,
+    sort_type: sortType,
+    [searchField]: searchQuery,
+  });
+
+  const { mutate: changeStatus } = useChangeStatusApi();
+
+  const handleOpenModal = (userId: string) => {
+    setSelectedUser({ user_id: userId, status: -1 });
+    setIsModalOpen(true);
+  };
+  const handleCloseModal = () => setIsModalOpen(false);
+
+  const handleClickAction = (userId: string, userStatus: UserStatusEnum) => {
+    setIsConfirmModalOpen(true);
+    setSelectedUser({ user_id: userId, status: userStatus === '1' ? 9 : 1 });
```

Comment:
same as above. Please apply all comment from RestaurantManagementTab to this tab